src/components/DocumentsView/DocumentView.tsx

```diff
@@ -27,10 +43,15 @@ export const DocumentView: FunctionComponent<IDocumentViewProps> = (props) => {
         <Card className="sbo-gray-gradiant">
             <Row className="m-3">
                 <Col>
-                    <a download href={props.document.LinkUrl}>
+                    <a download={!isOfficeFile} href={getDownloadUrl()}>
                         <Icon {...getFileTypeIconProps({ extension: extension, size: 20, imageFileType: 'png' })} className="show-overflow" />
                         <span className="align-middle">{' '}{props.document.Name}</span>
                     </a>
+                    {isOfficeFile &&
+                        <a download href={props.document.LinkUrl}>
+                            <span className="align-middle ml-3">Download</span>
+                        </a>
+                    }
```

Comment:
Needs some separation from the file name. Since they're both links, they sort of blend together. Maybe move the download link to the right side, or the bottom right corner of the card?
Can we get a download icon too?
<a><icon for download /> Download</a>